cmake/checkcxx11.cpp

```diff
@@ -11,8 +11,8 @@ class Ptr {
     inline Ptr(const Ptr<T>& ptr) : ptr_(ptr.ptr_) {}
     template<class T2> inline Ptr(const Ptr<T2>& ptr) : ptr_(ptr.get()) {}
     Ptr<T>& operator=(const Ptr<T>& ptr) { ptr_ = ptr.get(); return *this; }
-    inline Ptr(Ptr&&) = default;
-    Ptr<T>& operator=(Ptr&&) = default;
+//    Ptr(Ptr&&) = default;
+//    Ptr<T>& operator=(Ptr&&) = default;
```

Comment:
This file just reflects all the features that we need, so if we change our main code, we just do the corresponding change here.